src/menu/ImGuiImpl.cpp

```diff
@@ -202,17 +202,19 @@ void ImGuiWMInit() {
             ImGui_ImplWiiU_Init();
             break;
 #else
-        case Backend::SDL:
+        case Backend::SDL_OPENGL:
             SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "1");
-            if (impl.Metal.Window) {
-                ImGui_ImplSDL2_InitForMetal(static_cast<SDL_Window*>(impl.Metal.Window));
-                break;
-            }
-
             SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
             ImGui_ImplSDL2_InitForOpenGL(static_cast<SDL_Window*>(impl.Opengl.Window), impl.Opengl.Context);
             break;
 #endif
+#if __APPLE__
+        case Backend::SDL_METAL:
+            SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "1");
+            SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
```

Comment:
it'd be nice to not have these in 2 of the cases, but trying to not copy/paste would probably make this switch/case even messier